apps/docs/components/JwtGenerator.js

```diff
@@ -57,19 +56,16 @@ export default function JwtGenerator({}) {
       </div>
       <div className="grid mb-8">
         <label for="service">Preconfigured Payload:</label>
-        <select
-          id="service"
-          onChange={handleKeySelection}
-          className="border rounded bg-gray-200 w-full"
-        >
-          <option value="anon">ANON_KEY</option>
-          <option value="service">SERVICE_KEY</option>
-        </select>
+        <Select id="service" onChange={handleKeySelection}>
+          <Select.Option value="anon">ANON_KEY</Select.Option>
+          <Select.Option value="service">SERVICE_KEY</Select.Option>
+        </Select>
       </div>
 
       <div className="grid mb-8">
         <label for="token">Payload:</label>
         <textarea
```

Comment:
Will `<Input.TextArea />` work here instead of the custom input?

Reply:
I haven't tried but let me check